projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -220,6 +220,10 @@ public function __construct() {
 		add_action( 'init', array( $this, 'add_post_type_support' ) );
 		add_action( 'init', array( $this, 'register_post_meta' ), 20 );
 		add_action( 'jetpack_register_gutenberg_extensions', array( $this, 'register_gutenberg_extension' ) );
+
+		// The custom priority for this action allows a 'jetpack_publicize_force_featured_image_panel'
```

Comment:
As mentioned in my comment, I wonder if this prority should go the other way, say 8, to ensure that any code to remove support for thumbnails from a theme would still have an opportunity to run.

It might also mean that we don't need the `jetpack_publicize_force_featured_image_panel` because someone could call `remove_action` on `add_theme_post_thumbnails_support`.

Reply:
I think I understand the rationale. It means that we wouldn't break existing code that already calls `remove_theme_support( 'post-thumbnails' )`.

> It might also mean that we don't need the `jetpack_publicize_force_featured_image_panel` because someone could call `remove_action` on `add_theme_post_thumbnails_support`.

How easy is it to call `remove_action` when the callback is a member function? That might be some PHP foo I'm unaware of, but it seems to me that `remove_action` only works when the callback is a global function?

Reply:
Yes, it can be done, but it's difficult without a global reference. There is a global `$publicize` variable that should be the object we'd need, so we could document that, or we could make the function static, as it's not relying on anything else in the class.

Alternatively, all [this code is going to live in a namespace now](https://github.com/Automattic/jetpack/blob/master/projects/packages/publicize/src/class-publicize-base.php#L10), so we could have the function outside of the class instead. We are currently in the process of getting Jetpack to use this package instead, so if we go this route we could maybe wait for that, and adapt this change then.